web/pages/access.tsx

```diff
@@ -0,0 +1,21 @@
+import CreateAccountForm from "../components/CreateAccountForm";
+import AccountTable from "../components/AccountTable";
+import styles from "../components/AccessManagementPage.module.css";
+import { useState } from "react";
+
+export default function Access() {
+  const [accountList, updateAccountList] = useState<Object[] | null>([]);
```

Comment:
Use `IAccount[]` type here instead of `Object[]`. We want to be as specific as possible in our type checking!